client/public/stylesheets/index.css

```diff
@@ -0,0 +1,17 @@
+
+* {
+    background-color: #A7A8A5;
+    color: white;
+    font-family: Kallisto;
+
+}
```

Comment:
I'd recommend using `.scss` rather than `.css` files for the added "superpowers"!

rather than a wildcard here, we should apply styles directly of put "globally applied" styles in the `main.scss` for better organization. Then styles related to the index page can go here